tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-ortmodule-test-clear-cache-pipeline.yml

```diff
@@ -0,0 +1,25 @@
+trigger: none
```

Comment:
Does it need a `schedules:` entry in the YAML to configure the frequency? https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml

Reply:
not required over here. The schedule can be set up on the pipeline setup webpage. So, it is not required here.